packages/web-app-files/src/index.js

```diff
@@ -26,62 +38,69 @@ const navItems = [
     name: $gettext('All files'),
     iconMaterial: appInfo.icon,
     route: {
-      name: 'files-personal',
-      path: `/${appInfo.id}/list/all`
-    }
-  },
-  {
-    name: $gettext('Favorites'),
-    iconMaterial: 'star',
-    route: {
-      name: 'files-favorites',
-      path: `/${appInfo.id}/list/favorites`
-    },
-    enabled(capabilities) {
-      return capabilities.files && capabilities.files.favorites
+      path: `/${appInfo.id}/spaces/`
     }
   },
   {
     name: $gettext('Shared with me'),
     iconMaterial: 'shared-with-me',
     route: {
-      name: 'files-shared-with-me',
-      path: `/${appInfo.id}/list/shared-with-me`
+      path: `/${appInfo.id}/shares/with-me`
     }
   },
   {
     name: $gettext('Shared with others'),
     iconMaterial: 'shared-with-others',
     route: {
-      name: 'files-shared-with-others',
-      path: `/${appInfo.id}/list/shared-with-others`
+      path: `/${appInfo.id}/shares/with-others`
     }
   },
   {
     name: $gettext('Shared via link'),
     iconMaterial: 'link',
     route: {
-      name: 'files-shared-via-link',
-      path: `/${appInfo.id}/list/shared-via-link`
+      path: `/${appInfo.id}/shares/via-link`
     }
   },
   {
     name: $gettext('Deleted files'),
     iconMaterial: 'delete',
+    route: {
+      path: `/${appInfo.id}/trash`
+    },
     enabled(capabilities) {
       return capabilities.dav && capabilities.dav.trashbin === '1.0'
-    },
+    }
+  },
+  {
+    name: $gettext('Favorites'),
```

Comment:
Why did you move Favorites to the end? Please move back to original order. 😉

Reply:
Shiddo, this happened due refactoring where I duplicated and relocated the links. Good catch